source/_integrations/zha.markdown

```diff
@@ -99,6 +99,7 @@ The hardware-independent design of this integration provides support for many Zi
 ### Recommended Zigbee radio adapters and modules
 
 - Silicon Labs EmberZNet based radios using the EZSP protocol (via the [bellows](https://github.com/zigpy/bellows) library for zigpy)
+  - [Home Assistant Connect ZBT-2](/connect/zbt-2/) (EFR32MG24-based USB dongle)
```

Comment:
Maybe we should use "adapter" instead of dongle here. Both are fine with me.
```suggestion
  - [Home Assistant Connect ZBT-2](/connect/zbt-2/) (EFR32MG24-based USB adapter)
```